### DISCUSSION TITLE
Themed images don't work on marketplace

### DISCUSSION THREAD

**Discussion by Katsute** [community/community#8852] [Other Feature Feedback, Questions, & Ideas]
[Themed images](https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/) do not work on the [marketplace](https://github.com/marketplace) READMEs. Obviously the marketplace doesn't yet support dark mode, but I feel that for the time being it should hide any dark mode themed images.

**Comment by gaknoll** [comment]
Thanks for sharing this feedback @Katsute! I'll share it with the team who shipped that feature.
